Training/Api/GetResponseInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Training\Api;
+
+/**
+ * Interface GetResponseInterface
+ * @package Codifi\Training\Api
+ */
+interface GetResponseInterface
```

Comment:
Logical mistake in interface name as it's **response interface**, not getter for response. Also for interface name we use nouns, not verbs